cmd/license-info.go

```diff
@@ -19,15 +19,14 @@ package cmd
 
 import (
 	"fmt"
-	"strings"
 	"time"
 
 	"github.com/fatih/color"
+	"github.com/jedib0t/go-pretty/v6/table"
```

Comment:
we should avoid introducing newer libraries as deps - instead of this please use https://github.com/charmbracelet/bubbles#table

We already use this everywhere and standardizing on this for terminal UI.